packages/bee-q/src/components/dropdown/bq-dropdown.tsx

```diff
@@ -0,0 +1,164 @@
+import { h, Component, Element, Event, EventEmitter, Prop, Listen } from '@stencil/core';
+
+import { FloatingUIPlacement } from '../../services/interfaces';
+
+/**
+ * @part base - The component's internal wrapper.
+ * @part trigger - The `div` element used to display the trigger element
+ * @part panel - The `div` element used to display the panel element (bq-panel)
+ */
+@Component({
+  tag: 'bq-dropdown',
+  styleUrl: './scss/bq-dropdown.scss',
+  shadow: true,
+})
+export class BqDropdown {
+  // Own Properties
+  // ====================
+
+  private trigger: HTMLElement;
+  private panelElement: HTMLBqPanelElement;
+
+  // Reference to host HTML element
+  // ===================================
+
+  @Element() el!: HTMLBqDropdownElement;
+
+  // State() variables
+  // Inlined decorator, alphabetical order
+  // =======================================
+
+  // Public Property API
+  // ========================
+
+  /** Distance (px) between the panel and the trigger element. */
+  @Prop({ reflect: true }) panelDistance?: number = 4;
+
+  /** Position of the panel */
+  @Prop({ reflect: true }) panelPlacement?: FloatingUIPlacement = 'bottom-start';
+
+  /**
+   * If true, panel is visible.
+   * You can toggle this attribute to show/hide the panel.
+   */
+  @Prop({ reflect: true }) panelOpen?: boolean = false;
+
+  /**
+   * Determines whether the scrollbar is visible or hidden within the panel.
+   */
+  @Prop({ reflect: true }) panelScrollbar?: boolean = false;
```

Comment:
We need to allow the same properties that `bq-panel` should receive from the parent:

```suggestion
  /** Represents the distance (gutter or margin) between the dropdown panel and the trigger element. */
  @Prop({ reflect: true }) distance?: number = 4;

  /** If true, the dropdown panel will be visible. */
  @Prop({ reflect: true, mutable: true }) open?: boolean = false;

  /** Position of the dropdown panel */
  @Prop({ reflect: true }) placement?: FloatingUIPlacement = 'bottom';

  /** Whether the dropdown panel should have the same width as the trigger element */
  @Prop({ reflect: true }) sameWidth?: boolean = false;

  /**  Represents the skidding between the dropdown panel and the trigger element. */
  @Prop({ reflect: true }) skidding?: number = 0;

  /** Defines the strategy to position the dropdown panel */
  @Prop({ reflect: true }) strategy?: 'fixed' | 'absolute' = 'fixed';
```